hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileCreateRequest.java

```diff
@@ -194,6 +212,28 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
             OMException.ResultCodes.NOT_A_FILE);
       }
 
+      // Check if Key already exists in KeyTable and this transaction is a
+      // replay.
+      String ozoneKey = omMetadataManager.getOzoneKey(volumeName, bucketName,
+          keyName);
+      OmKeyInfo dbKeyInfo = omMetadataManager.getKeyTable().get(ozoneKey);
+      if (dbKeyInfo != null) {
+        // Check if this transaction is a replay of ratis logs.
+        // Check if this transaction is a replay of ratis logs.
```

Comment:
Minor: Duplicate  // Check if this transaction is a replay of ratis logs.
We can remove one.